sycl/source/handler.cpp

```diff
@@ -1464,6 +1464,53 @@ checkContextSupports(const std::shared_ptr<detail::context_impl> &ContextImpl,
   return SupportsOp;
 }
 
+void handler::verifyDeviceHasProgressGuarantee(
+    sycl::ext::oneapi::experimental::forward_progress_guarantee Guarantee,
+    sycl::ext::oneapi::experimental::execution_scope threadScope,
```

Comment:
to align naming with other parameters

Reply:
it is just a NIT but idea was to do just the opposite. see other functions (e.g. line 1457).